app/views/static_pages/community_expectations.html.erb

```diff
@@ -1,11 +1,87 @@
 <%= title('Community Expectations') %>
-<% filename = 'app/views/static_pages/dashboard_steps/community_expectations.md' %>
+<div class="gradient py-8">
+  <div class="col-xl-8 offset-xl-2 text-gray-500">
+
+    <h2 class="font-bold text-2xl py-4">Community Expectations</h2>
+
+    <p class="mb-4">
+      The following expectations are important of the culture in our Discord community. They are not rules that are enforced by our moderation team, but they will help you get the most out of your experience in our Discord server.
+    </p>
+
+    <h3 class="font-bold text-xl mt-7 mb-5">1. Ask detailed and specific questions</h3>
+
+    <p class="my-4">
+      We encourage questions, so do not worry about asking a “stupid question”. However, it is important that you know <a href="/before_asking" class="text-gold-600 underline decoration-solid" target="_blank">How to Help Yourself Before Asking Others</a> and follow our tips on <a href="/how_to_ask" class="text-gold-600 underline decoration-solid" target="_blank">How to Ask a Technical Question</a>. In addition, to ensure that everyone has the same expectations, below are some specific guidelines on asking questions:
+    </p>
+
+    <div class="card-main">
```

Comment:
With the prose classes applied, I think this looks and flows better without card styling
<img width="731" alt="Screenshot 2022-09-11 at 14 24 19" src="https://user-images.githubusercontent.com/7963776/189530059-e5a97237-a2a9-4f47-b1d6-277a5827177c.png">




Same for the other lists that are using card-main below.